content/activities/StreamingAPodcast.ts

```diff
@@ -0,0 +1,12 @@
+import Activitiy from '@/types/Activity'
+
+const StreamingAPodcast: Activitiy = {
+  title: 'Streaming A Podcast',
+  imagePath: 'images/podcast.jpg',
+  alt: 'Podcast サイトの一覧画面です',
+  content: 'Podcastで所属メンバーが技術のことから暮らしのことまで語っています',
+  linkTitle: 'Podcast サイト',
+  url: 'https://oystersjp.github.io/podcast/'
```

Comment:
```suggestion
  url: '/podcast'
```

[nits]ドメイン買うかもしれんから・・

Reply:
あ、確かにここフルパスじゃなくても遷移できるか！
そうですね、そうしましょ！